assembly-combined-package/assembly-combined/conf/application-linkis.yml

```diff
@@ -35,4 +35,14 @@ pagehelper:
 
 #ribbon:
 #  ReadTimeout: 10000
-#  ConnectTimeout: 10000
\ No newline at end of file
+#  ConnectTimeout: 10000
+
+#  enable for knife4j used by all micro services except gateway and eureka
+#knife4j:
+#  enable: true
+#  basic:
+#    enable: false
+#    # Basic认证用户名
+#    username: test
+#    # Basic认证密码
+#    password: 123
```

Comment:
If a password is used, the password should not be too simple，eg:#1105.

Reply:
ok,I will remove this username/password